appserver/ejb/ejb-full-container/src/main/java/org/glassfish/ejb/mdb/deployment/annotation/handlers/MessageDrivenHandler.java

```diff
@@ -142,6 +145,26 @@ protected HandlerProcessingResult setEjbDescriptorInfo(
             // xml override
             if (acProp.propertyName().equals("resourceAdapter")) {
                 ejbMsgBeanDesc.setResourceAdapterMid(envProp.getValue());
+            } else if (acProp.propertyName().equals("MaxPoolSize")) {
```

Comment:
The default value for these properties is defined in `appserver/deployment/dol/src/main/java/com/sun/enterprise/deployment/DescriptorConstants.java`. 
String constants should also be added for these properties name.